packages/amplify-app/src/scripts/amplify-push.js

```diff
@@ -89,26 +89,26 @@ async function run() {
   }
 
   const PROJECT_CONFIG = `{\
-    \"envName\":\"${buildConfig.envName || 'amplify'}\"\
+    "envName":"${buildConfig.envName || 'amplify'}"\
   }`;
 
   let PROVIDER_CONFIG;
 
   if (buildConfig.profile) {
     PROVIDER_CONFIG = `{\
-      \"awscloudformation\": {\
-        \"configLevel\":\"project\",\
-        \"useProfile\":true,\
-        \"profileName\":\"${profileToUse}\"\
+      "awscloudformation": {\
```

Comment:
Don't we need to quote and escape as this is passed as JSON string to the a child process

Reply:
Tested manually and scripts were working perfectly, no extra escaping required.